racket/collects/racket/match/define-forms.rkt

```diff
@@ -16,8 +16,9 @@
 (provide define-forms)
 
 (define-syntax-rule (define-forms parse-id
-                      match match* match-lambda match-lambda*
-		      match-lambda** match-let match-let*
+                      match match* match-lambda match-λ 
```

Comment:
Is it possible to use `(rename-out match-lambda match-λ)` etc here? That would be simpler than using `define-syntaxes`

Reply:
I think `match-λ` here is correct already. `rename-out` should be used below instead (in `provide`).

@jasonhemann do you still want to make the change?

Reply:
@sorawee no, please feel free.

Reply:
> I think `match-λ` here is correct already. `rename-out` should be used below instead (in `provide`).
> 
> @jasonhemann do you still want to make the change?
@sorawee no, please feel free.

Reply:
In that case, I propose that we close this PR. Is that OK?